lib/torrent.js

```diff
@@ -921,7 +921,7 @@ class Torrent extends EventEmitter {
       this._downloadSpeed(downloaded)
       this.client._downloadSpeed(downloaded)
       this.emit('download', downloaded)
-      this.client.emit('download', downloaded)
+      if (this.client) this.client.emit('download', downloaded)
```

Comment:
I would rather use:
```suggestion
      if (this.destroyed) return
      this.client.emit('download', downloaded)
```
Just to be consistent with the rest of the code. (same thing with the other change)

Reply:
suggestion applied with https://github.com/webtorrent/webtorrent/pull/1925/commits/4a5597149541709a1a95bcd46bfbb7efcf3b4102